data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -67,6 +72,93 @@ then
 	touch cvo-bootstrap.done
 fi
 
+# We originally wanted to run the etcd cert signer as
+# a static pod, but kubelet could't remove static pod
+# when API server is not up, so we have to run this as
+# podman container.
+# See https://github.com/kubernetes/kubernetes/issues/43292
+
+echo "Starting etcd certificate signer..."
+
+trap "podman rm --force etcd-signer" ERR
+
+bootkube_podman_run \
+	--name etcd-signer \
+	--detach \
+	--volume /opt/openshift/tls:/opt/openshift/tls:ro,z \
+	"${KUBE_ETCD_SIGNER_SERVER_IMAGE}" \
+	serve \
+	--cacrt=/opt/openshift/tls/etcd-signer.crt \
+	--cakey=/opt/openshift/tls/etcd-signer.key \
+	--metric-cacrt=/opt/openshift/tls/etcd-metric-signer.crt \
+	--metric-cakey=/opt/openshift/tls/etcd-metric-signer.key \
+	--servcrt=/opt/openshift/tls/kube-apiserver-lb-server.crt \
+	--servkey=/opt/openshift/tls/kube-apiserver-lb-server.key \
+	--servcrt=/opt/openshift/tls/kube-apiserver-internal-lb-server.crt \
+	--servkey=/opt/openshift/tls/kube-apiserver-internal-lb-server.key \
+	--servcrt=/opt/openshift/tls/kube-apiserver-localhost-server.crt \
+	--servkey=/opt/openshift/tls/kube-apiserver-localhost-server.key \
+	--address=0.0.0.0:6443 \
+	--insecure-health-check-address=0.0.0.0:6080 \
+	--csrdir=/tmp \
+	--peercertdur=26280h \
+	--servercertdur=26280h \
+	--metriccertdur=26280h
+
+if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ] && [ ! -f etcd-bootstrap.done ]
+then
+	echo "Rendering etcd Operator Manifests..."
+	rm -rf etcd-bootstrap
+	mkdir -p /etc/etcd
+	bootkube_podman_run \
+		--volume "$PWD:/assets:z" \
+		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+		/usr/bin/cluster-etcd-operator render \
+		--etcd-ca=/assets/tls/etcd-ca-bundle.crt \
+		--etcd-metric-ca=/assets/tls/etcd-metric-ca-bundle.crt \
+		--manifest-etcd-image "${MACHINE_CONFIG_ETCD_IMAGE}" \
+		--etcd-discovery-domain {{.ClusterDomain}} \
+		--manifest-cluster-etcd-operator-image "${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+		--manifest-setup-etcd-env-image "${MACHINE_CONFIG_OPERATOR_IMAGE}" \
+		--manifest-kube-client-agent-image "${MACHINE_CONFIG_KUBE_CLIENT_AGENT_IMAGE}" \
+		--asset-input-dir /assets/tls \
+		--asset-output-dir /assets/etcd-bootstrap \
+		--config-output-file /assets/etcd-bootstrap/config
+fi
+
+if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+then
+	# during initial operator rollout phase this logic allows us to deploy the operator via CVO in an `Unmanaged` no-op state. after all of the pieces have
+	# merged and the operator is deemed stable we can remove this logic and the operator will be `Managed` by default.
+	CLUSTER_ETCD_OPERATOR_MANAGED=$(bootkube_podman_run \
```

Comment:
Wondering if we can move this if condition to https://github.com/openshift/installer/pull/2730/files#diff-ce82c1d8a44f7dfc41dfc024085ccfeeR25, set `CLUSTER_ETCD_OPERATOR_MANAGED` at the very start and use this globally instead of `CLUSTER_ETCD_OPERATOR_IMAGE `? 

Just a thought to improve readability, not sure if it would be possible or not!

Reply:
I think its a nice idea. But because the logic is temporary until we basically merge the operator and it is stably scaling. I would prefer just keep it as a small removable block where it is. If it were not temporary I think a more creative solution would make sense. This way we dont have to change anything else except moving the copy files up to the main if statement when we move to Managed.

Reply:
I ended up doing something similar, thanks for the input.